app/components/Location.tsx

```diff
@@ -0,0 +1,23 @@
+import { Box } from "@chakra-ui/react";
+
+const content = [
+  "Our family-owned CNC production company is located in Duvall, Washington, just 30 miles northeast of the Seattle metropolitan area. Clients of Kenworthy Machine appreciate our local accessibility as well as our statewide service and delivery.",
+];
+
+const Location = () => {
+  return (
+    <Box
+      as="p"
+      bg="brand.backgroundMed"
+      fontSize={{ base: "2rem", xl: "3rem" }}
+      fontWeight="bold"
+      lineHeight={{ base: "2.4rem", xl: "3.6rem" }}
+      p={{ base: "3.125rem 2.5rem", xl: "8rem 4.5rem" }}
+      textAlign={"center"}
+    >
+      {content}
```

Comment:
Should we add max widths as we discussed in our meeting? Currently the content stretches as much as you stretch the browser screen. 
You might need a nested `Box` around the content to add the `maxW` property. In Figma it's 1562 pixels, you could round to 1550 or 1560, see how it looks proportionate to the design and adjust. (I used the Figma design width to set a max width for the values section, but it might be too wide).

Reply:
Sure, I could do that, but for some reason I thought that it would be easier to set up for the whole page rather than individual components. I'll play around with it to see if it's possible

Reply:
I am thinking the same as Alex on this one. If we want to give the site a maxWidth for all pages, it should be done at the RootLayout level, probably using grid template columns. If we want this to apply to only specific components, I think we can create a reusable layout component that can be imported as a component wrapper. Maybe see [this link](https://nextjs.org/docs/app/building-your-application/routing/pages-and-layouts) also.

Reply:
I've added `Box` component with `max-width` and `mx` props in `layout.tsx` around the `{children}`. I'm not sure if footer needs to be included in that as well or not; to me it looks a bit funky stretched full-width.

Reply:
Also, I just realized that I set max-width to 1500px, because that's what @avinashi10 was talking about above. But, since the original Figma design was created for 1728px screen, should the max-width be closer to that number instead? I'm getting more and more confused with sizing adjustments to be honest

Reply:
I was thinking of an AppWrapper that would include the header and footer too. And, yes, according to the design, the width of the MacBook pro 16" is 1728, so everything should max out at that.